### ISSUE TITLE
Fix tests

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/publish.webmaker.org#196]
The tests right now seem to have quite a number of issues with it (other than the fact that it is really hard to follow because of its complicated architecture). Let's identify these issues (one of them was fixed in https://github.com/mozilla/publish.webmaker.org/commit/8f37da04ff54f926bff7d75553a0c0bab46e26f6), file follow-up bugs, and fix them.

In short, the tests need to be re-written and refactored.

**Comment by sedge** [comment]
+1

**Comment by gideonthomas** [comment]
Also change the tokens and seeds that we use for testing to be more generic like `test` or something similar so that we don't include real usernames.
